generators/azure-spring-cloud/generator.mjs

```diff
@@ -283,7 +285,7 @@ ${chalk.red('az extension add --name spring-cloud')}`
         statistics.sendSubGenEvent('generator', GENERATOR_AZURE_SPRING_CLOUD);
       },
 
-      azureSpringCloudAppCreate() {
+      azureSpringCloudAppCreate: runAsync(function () {
```

Comment:
sceptic compared to async/await

Reply:
Yes, ideally we should migrate to await/async or promises.
Unfortunately we have a lot of functions that uses this old feature.
 - It's a legacy feature and will be removed from generator v6.
 - Migrating to await/async or promises takes time to test.
 - Each generator runs specific commands with specific knowledges.